config/locales/activerecord.en.yml

```diff
@@ -0,0 +1,125 @@
+---
+en:
+  activerecord:
+    attributes:
+      client:
+        city: City
+        comments: Comments
+        competent_authority: Competent authority
+        date_of_birth: Date of birth
+        education: Education
+        first_name: First name
+        gender: Gender
+        goals: Goals
+        hobbies: Hobbies
+        relatives: Family members
+        language_skills: Language skills
+        interests: Interests
+        involved_authority: Involved authority
+        last_name: Last name
+        nationality: Nationality
+        permit: Permit
+        phone: Phone
+        state: State
+        street: Street
+        zip: Zip
+        registered_by: Responsible Volunteer
```

Comment:
I was just going back and forth with getting them where I needed them. Very possible that they're redundant in some cases, but certainly not in all. And do they hurt?